example/stop-resume/index.html

```diff
@@ -0,0 +1,55 @@
+<head>
+  <style> body { margin: 0; } </style>
+
+  <script src="//unpkg.com/3d-force-graph"></script>
+  <!-- <script src="../../dist/3d-force-graph.js"></script> -->
+</head>
+
+<body>
+  <button id="stop-resume" style="display: block; margin: 20px; height: 30px; width: 100px;">Stop/Resume</button>
```

Comment:
Perhaps the button's text can toggle according to the animation state, i.e. say `Stop animation` when it's running, and `Resume animation` when it's not.